python/lsst/ts/standardscripts/maintel/m1m3/enable_m1m3_balance_system.py

```diff
@@ -0,0 +1,72 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["EnableM1M3BalanceSystem"]
+
+import time
+
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+
+from ...base_block_script import BaseBlockScript
+
+
+class EnableM1M3BalanceSystem(BaseBlockScript):
+    """Enable M1M3 force balance system.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    - "Enabling M1M3 force balance system": Before enabling M1M3 force balance
+    system.
+
+    **Details**
+
+    This script enables the M1M3 force balance system of the Simonyi Main
+    Telescope.
+
+
+    """
+
+    def __init__(self, index, add_remotes: bool = True):
+        super().__init__(index=index, descr="Enable M1M3 force balance system")
+
+        mtcs_usage = None if add_remotes else MTCSUsages.DryTest
+
+        self.mtcs = MTCS(self.domain, intended_usage=mtcs_usage, log=self.log)
```

Comment:
Check the comment I left in the other script about using `configure_tcs`

Reply:
By other script, do you mean defining a configure_tcs method and calling await configure_tcs in the __init__?
I guess sort of like here: https://github.com/lsst-ts/ts_standardscripts/blob/0c6ea98ec04a251644d9d96f6b91d5d17cccefde/python/lsst/ts/standardscripts/maintel/move_p2p.py#L46-L56

Reply:
Oh! Just ignore this comment. I wrote a larger comment before this one but honestly, I don't think it is worth the hassle. I think you van just rebase and fix the version history and we're good.